media/static-bundles.json

```diff
@@ -740,6 +740,12 @@
         "css/firefox/unfck.de.scss"
       ],
       "name": "firefox-unfck-de"
+    },
```

Comment:
Not that I know of. Though for this one I don't know if this should get its own bundle. A better approach may be to either make it a Sass partial to import into pages that need it, OR add it as another file in the individual CSS bundle for each page that uses it. The JS for the promo can also be added to page bundles.

On a third hand, if we want to control the promo with a waffle switch it might need a dedicated bundle after all.